### PR TITLE
Add alias treatment to method_documentation.rdoc

### PR DESCRIPTION
**Pull request by BurdetteLamar**

### CODE REVIEW

**Comment by BurdetteLamar** [comment]
This is mainly motivated by the messy potential around Array#[] and Array#slice.

File: doc/method_documentation.rdoc [hunk 1]
```diff
@@ -62,6 +63,12 @@ Instead of:
 However, as shown above for <tt>Array#count</tt>, use separate lines if the
 behavior is different if the argument is omitted.
 
+Omit aliases from the call-seq.
+
+== Aliases
+
+Mention aliases in the form "Array#find_index is an alias for Array#index."
+- "
```
Comment (jeremyevans) [inline doc/method_documentation.rdoc:71]: I think this line should be blank.

Comment (BurdetteLamar) [reply doc/method_documentation.rdoc:71]: You so right.

**Comment by BurdetteLamar** [comment]
> 
> 
> I think any aliases can be mentioned after the details and examples, or at least after the synopsis.

Maybe before Related, if any, else last?

**Comment by jeremyevans** [comment]
> > I think any aliases can be mentioned after the details and examples, or at least after the synopsis.
> 
> Maybe before Related, if any, else last?

That works for me.

**Pull request merged**